dev-tools/jenkins_release.sh

```diff
@@ -16,4 +16,10 @@ cleanup() {
 }
 trap cleanup EXIT
 
+# This controls the defaults used the Jenkins package job. They can be
+# overridden by setting them in the environement prior to running this script.
+export BEAT_VERSION_QUALIFIER="${VERSION_QUALIFIER:-alpha1}"
```

Comment:
So these values would be adjusted based on the branch?

Reply:
Yes, that would be the case. The alternative would be to put it in the JJB config in infra, but we want to get away from that model in order to have more control over our Jenkins jobs from our repo.